### PR TITLE
feat: added the github repo link to the nav

### LINKED ISSUE #12: [FEATURE] Add the github repo link to the nav
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

Hello:)
I think we should let people know where is the source code.

### Proposed Solution

Add a github link on the header

### Screenshots

_No response_

### Do you want to work on this issue?

No

### If "yes" to above, please explain how you would technically implement this (issue will not be assigned if this is skipped)

_No response_

### PR DESCRIPTION
**Pull request by byhlel**
Resolves #12

## Description

added the github repo link to the nav

New dependency: React Icons, we can use Font Awesome, Material Icons if you prefer

Fixes #12 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

## Screenshots (if applicable):
image

### CODE REVIEW

File: src/components/Nav.tsx [hunk 1]
```diff
@@ -15,6 +16,9 @@ const Navbar: React.FC = () => {
         <Link href="/" className="uppercase">
           Trend Track
         </Link>
+        <Link href="https://github.com/Bashamega/TrendTrack" target="_blank">
```
Comment (Bashamega) [inline src/components/Nav.tsx:19]: make it near the api text. You can achieve this by wrapping the github icon and the api link in one div

**Review by Bashamega — CHANGES_REQUESTED** [review]

File: src/components/Nav.tsx (hunk 1 above)
Comment (byhlel) [reply src/components/Nav.tsx]: done

File: src/components/Nav.tsx [hunk 2]
```diff
@@ -39,12 +40,17 @@ const Navbar: React.FC = () => {
         <div
           className={`md:flex space-x-4 ${isOpen ? "block" : "hidden"} md:block`}
         >
+        <div className="flex space-x-4 items-center">
+          <Link href="https://github.com/Bashamega/TrendTrack" target="_blank">
+            <FaGithub/>
```
Comment (Bashamega) [inline src/components/Nav.tsx:45]:
```suggestion
            <FaGithub/> Github
```

**Review by Bashamega — APPROVED** [review]

**Comment by Bashamega** [comment]
Great work. thanks

**Pull request merged**